platform/rct/Project/STM32_FLASH.ld

```diff
@@ -36,8 +36,8 @@ ENTRY(Reset_Handler)
 _estack = 0x20000000 + 40K;    /* end of 40K RAM */
 
 /* Generate a link error if heap and stack don't fit into RAM */
-_Min_Heap_Size = 0x200;      /* required amount of heap  */
-_Min_Stack_Size = 0x400; 	/* required amount of stack */
+_Min_Heap_Size = 0x100;      /* required amount of heap  */
+_Min_Stack_Size = 0x100; 	/* required amount of stack */
```

Comment:
Ahh... sorry misinterpreted the variable names.  You are right and this is sane.  I wish we could recover all of that space.  Not sure how we would go about doing that but there is probably a way...